### ISSUE TITLE
A Daughter's Embrace

### ISSUE THREAD

**Issue by KevlinTallfellow** [gamefreedomgit/Maelstrom#5177]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (http://wowhead.com/)
[//]: # (cata-twinhead.twinstar.cz)

**Description:**

Lilian Voss is supposed to accompany the player to the tower where an event takes place.  Even when I somehow managed to keep myself and her alive while she attacked every single enemy along the road, when we finally walked into the tower, nothing happened.  No event triggered.  Even after dying, running back to the tower, and respawning inside to have her appear next to me, no event triggered.  I tried moving outside and back in to cross any triggers there might be at the tower, to no avail.

**How to reproduce:**

Obtain the quest.  Have Lilian follow you into the tower, probably after clearing away any of the NPCs that she attacks outside of the tower.

**How it should work:**
When Lilian follows the player into the tower, an event should trigger where she talks to her father and subsequently kills him.

**Database links:**

https://www.wowhead.com/quest=25046/a-daughters-embrace#comments

**Comment by KevlinTallfellow** [comment]
This quest is necessary to continue the quest series leading to Silverpine, so until it is fixed the entire rest of the Tirisfal Glades quest series is unavailable.

Also, when I got to the tower, the quest NPCs were already in the tower, seemingly broken.  I was able to target them, and they were flagged as hostile, but they were all non-interactive.  These were High Priest Benedictus Voss, Captain Melrache, and Scarlet Bodyguard.  In the place of Benedictus Voss, there was an alternate version of him clipping through the first, as they were in exactly the same location but their idle animations were out of sync.  I was unable to target the alternate version of Voss, even using the /target command from different sides of his location.
